packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js

```diff
@@ -264,6 +272,7 @@ export default (async function PgIntrospectionPlugin(
       introspectionResultsByKind.type,
       "id"
     );
+    //TODO maybe just add these straight to the class objects
```

Comment:
https://github.com/graphile/graphile-build/blob/70f878364abb6e4d42c62026bf3a6fe7219c022b/packages/graphile-build-pg/src/plugins/PgIntrospectionPlugin.js#L390-L394 We do this here, now. We can't remove this though. Also the code in this link should (but doesn't) sort by attr.num.